ui/app/registration/directives/healthFacilityAddressFields.js

```diff
@@ -0,0 +1,206 @@
+'use strict';
+angular.module('bahmni.registration')
```

Comment:
@Edgencio have you included unit tests for these new functions and directive?

Reply:
Nop... i'll start working on that

Reply:
Thanks, maybe before you get too stuck in - 

@joaomachiana I recommend that we include unit tests with our changes it is good so we make sure we are producing robust and reliable code, but it also adds some extra time to the development. 

So we can either include now, which I think is best, and take a bit more time, or we start creating technical debt tickets so we can come back after and add the tests. What do you guys think?